stan/math/prim/scal/meta/contains_fvar.hpp

```diff
@@ -3,25 +3,16 @@
 
 #include <stan/math/prim/scal/meta/is_fvar.hpp>
 #include <stan/math/prim/scal/meta/scalar_type.hpp>
+#include <stan/math/prim/scal/meta/or.hpp>
 
 namespace stan {
 
 /**
- * Metaprogram to calculate the base scalar return type resulting
- * from promoting all the scalar types of the template parameters.
+ * Metaprogram to determine if any of the
```

Comment:
This comment seemed false to me (very similar to the /prim/scal/meta/return_type comment). I am not sure if this is descriptive enough.

Reply:
Yes, this is good.  You just want to reformulate the doc so that you're not describing the function, but saying what it returns.  So this one should be something like:

> Defines a public `enum` named `value` which is defined to be true (1) 
if any of the template parameters includes a `var` as their base scalar and false (0) otherwise.
```